Clarify which name to put in devices

heytcass opened the pull request:
When setting up Automatic, I found it confusing which name to put in the `devices` section, and put the nickname in, which broke the integration. Added a note to clarify for future users.

**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

heytcass committed:
Clarify which name to but in devices

When setting up Automatic, I found it confusing which name to put in the `devices` section, and put the nickname in, which broke the integration. Added a note to clarify for future users.

frenck commented on source/_integrations/automatic.markdown:
```diff
@@ -52,6 +52,11 @@ devices:
 {% endconfiguration %}
+<div class='note'>
+  
+The device name must be the name given by Automatic -ahem- *automatically*. This is typically the model year, make and model. This is *not* the nickname you have give the vehicle in the `vehicles` section of the app's settings.
```
Typo, and no need to mock Automatic in our documentation.

```suggestion
The device name must be the name given by Automatic automatically. This is typically the model year, make and model. This is *not* the nickname you have to give the vehicle in the `vehicles` section of the app's settings.
```

frenck commented on source/_integrations/automatic.markdown:
```diff
@@ -52,6 +52,11 @@ devices:
   required: false
   type: list
 {% endconfiguration %}
+<div class='note'>
```
Added newline to keep things nice & neat.
```suggestion

<div class='note'>
```

frenck reviewed (changes requested):
Thanks for the PR @heytcass.
Left 2 small suggestions, could you take a look? 👍

heytcass committed:
Resolve @frenck suggestions.

Yeah, I figured the tone was a bit too playful. Figured I'd give it a go though.

Co-Authored-By: Franck Nijhof <frenck@frenck.nl>

heytcass committed:
Resolve @frenck suggestions.

Co-Authored-By: Franck Nijhof <frenck@frenck.nl>

heytcass commented:
@frenck Resolved. 👍

frenck reviewed (approved):
Thanks for updating @heytcass! 👍
